modules/nw-ovn-kubernetes-list-resources.adoc

```diff
@@ -96,4 +94,18 @@ $ oc get pods ovnkube-node-jg52r \
 ovn-controller ovn-acl-logging kube-rbac-proxy kube-rbac-proxy-ovn-metrics ovnkube-node
 ----
 +
-The `ovnkube-node` pod has a container (`ovn-controller`) that resides on each {product-title} node. Each node’s `ovn-controller` connects the OVN northbound to the OVN southbound database to learn about the OVN configuration. The `ovn-controller` connects southbound to `ovs-vswitchd` as an OpenFlow controller, for control over network traffic, and to the local `ovsdb-server` to allow it to monitor and control Open vSwitch configuration.
\ No newline at end of file
+The `ovnkube-node` pod has a container (`ovn-controller`) that resides on each {product-title} node. Each node’s `ovn-controller` connects the OVN northbound to the OVN southbound database to learn about the OVN configuration. The `ovn-controller` connects southbound to `ovs-vswitchd` as an OpenFlow controller, for control over network traffic, and to the local `ovsdb-server` to allow it to monitor and control Open vSwitch configuration.
+
+. List the OVN-Kubernetes elected leader by running the following command:
```

Comment:
@tssurya Does this stmt seems ok to you? Or we need something like kubernetes elected OVN leader?

rest lgtm

Reply:
```suggestion
. List the current OVN-Kubernetes master leader pod by running the following command:
```